### PR TITLE
Minor bugs fix

### PR DESCRIPTION
**Pull request by novasdream**

### CODE REVIEW

File: data/monster/dreamhaunters/shock head.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
 		<immunity invisible="1"/>
 	</immunities>
 	<voices interval="5000" chance="10">
-		<voice sentence="<brrzz>"/>
```
Comment (att3) [inline data/monster/dreamhaunters/shock head.xml]: Why remove these <  >?

**Comment by TheSumm** [comment]
Your pull requests won't be merged if your repository is outdated.
I won't review any of the files until that is fixed.

**Comment by novasdream** [comment]
@TheSumm  I pull my repository before make that commit.

https://github.com/novasdream/Global-TFS/commit/b39b3fedf6cc50bb0b03b353e0048871a545fbb8

Why it dont work ?

File: data/monster/dreamhaunters/shock head.xml (hunk 1 above)
Comment (novasdream) [inline data/monster/dreamhaunters/shock head.xml]: Issue with unescape XML while in parsing files to search errors duplicated items or wrong information in monsters.

**Pull request closed without merging**